File: play.liracer.org/snippet/c/fizzbuzz.c
#include <stdio.h>

int main() {
	for(int i = 1; i <= 100; i++) {
		int fizzedOrBuzzed = 0;
		if(i % 3 == 0) {
			fizzedOrBuzzed = 1;
			printf("Fizz");
		}
		if(i % 5 == 0) {
			fizzedOrBuzzed = 1;
			printf("Buzz");
		}
		if(fizzedOrBuzzed) {
			printf("\n");
		} else {
			printf("%d\n", i);
		}
	}
}

File: play.liracer.org/snippet/go/fizzbuzz.go
//go:build ignore
package main

import "fmt"

func main() {
	for i := 0; i <= 100; i++ {
		fizzedOrBuzzed := false
		if i%3 == 0 {
			fizzedOrBuzzed = true
			fmt.Print("Fizz")
		}
		if i%5 == 0 {
			fizzedOrBuzzed = true
			fmt.Print("Buzz")
		}
		if !fizzedOrBuzzed {
			fmt.Print(i)
		}
		fmt.Println()
	}
}
